GVFS/GVFS.Installer/Setup.iss

```diff
@@ -165,6 +165,7 @@ DestDir: "{app}"; Flags: ignoreversion; Source:"{#ServiceDir}\GVFS.Service.exe";
 [UninstallDelete]
 ; Deletes the entire installation directory, including files and subdirectories
 Type: filesandordirs; Name: "{app}";
+Type: filesandordirs; Name: "{sd}\ProgramData\GVFS\GVFS.Upgrade";

```

Comment:
I'm curious, what does `{sd}` mean?

Reply:
System drive.

I think we should use `{commonappdata}` instead based on [this SO post](https://stackoverflow.com/questions/39655574/inno-setup-constant-for-programdata-directory?rq=1).